FirebaseAuth/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
-Fix a bug where the linking/reauth flows errors are not caught. (#10407)
+- [fixed] Fix a bug where the linking/reauth flows errors are not caught. (#10407)
```

Comment:
If there's an issue number, please use that instead of the PR. #10267 in this case?

Reply:
Yeah, make sense.